webpack/assets/javascripts/react_app/components/FactCharts/FactChart.fixtures.js

```diff
@@ -0,0 +1,36 @@
+import Immutable from 'seamless-immutable';
+import { STATUS } from '../../constants';
+import { mockStoryData } from '../common/charts/DonutChart/DonutChart.fixtures';
+
+export const chartDataValues = mockStoryData.data.columns;
+
+const state = {
+  modalToDisplay: {},
```

Comment:
in my last comment i meant set the state object to `Immutable` and use all of the others as you just did :)
```
const state = Immutable({ .... })
```

Reply:
Removed state and only use now initialState as the name initialState makes more sense here